x/evm/types/config.go

```diff
@@ -2,16 +2,18 @@ package types
 
 // EVMConfig contains all EVM module configuration values
 type EVMConfig struct {
-	Name       string `mapstructure:"name"`
-	RPCAddr    string `mapstructure:"rpc_addr"`
-	WithBridge bool   `mapstructure:"start-with-bridge"`
+	Name             string `mapstructure:"name"`
+	RPCAddr          string `mapstructure:"rpc_addr"`
+	WithBridge       bool   `mapstructure:"start-with-bridge"`
+	RPCModuleSupport bool   `mapstructure:"rpc-module-support"`
 }
 
 // DefaultConfig returns a configuration populated with default values
 func DefaultConfig() []EVMConfig {
 	return []EVMConfig{{
-		Name:       "Ethereum",
-		RPCAddr:    "http://127.0.0.1:7545",
-		WithBridge: true,
+		Name:             "Ethereum",
+		RPCAddr:          "http://127.0.0.1:7545",
+		WithBridge:       true,
+		RPCModuleSupport: true,
```

Comment:
```suggestion
		RPCModuleSupport: false,
```
this should default to `false` so that we don't need to set it for most of the nodes other than `ganache`.